script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -1022,13 +1022,13 @@ The first one is that they must be wealthy.
 [sha:83380ef9ab7b6e8944a05c12a6d5112e9d61e885]{
 -- Page 732, Offset 26260.
 -- 魔術のような学んで修得する神秘ではなく、生まれついてその肉体が持ってしまう特異能力。
-I'm not talking about mysteries you can learn, like magecraft,
+I'm not talking about the Arcane which you can study, like magecraft,
```

Comment:
Feels like this is an instance for Arcana?
> ...not talking about Arcana which you can study...

Reply:
Sure